lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -541,10 +541,8 @@ def __init__(self, canvas, window):
 
         NavigationToolbar2.__init__(self, canvas)
 
-    @cbook.deprecated("3.3")
-    @property
-    def ctx(self):
-        return self.canvas.get_property("window").cairo_create()
```

Comment:
To be fair, this feels easier to read with the usual decorator notation. If you really wanted to do this just to remove visual noise, maybe it makes sense if you instead put it above `__init__` where we normally put class variable definitions?

Reply:
In my view this is mostly useful in cases such as texmanager or widgets, where I feel that just stashing all the deprecated properties into a single block with no blank lines in between makes it easier to ignore that block.  (You immediately see where it starts and ends.)

Reply:
As below, I would be happy with reverting this one or just moving it above `__init__`, like we typically do for things that are defined as class variables.